INTRODUCTION.md

```diff
@@ -794,6 +794,64 @@ error log is silenced (will only be seen with debug enabled).
 but it is recommended to instead rely on the above heuristics.
 
 
+#### Sparse connections
+
+Maintaining an open connection to each broker in the cluster is problematic
+for clusters with a large number of brokers, as well with clusters with a
+large number of clients.
+
+To alleviate this the `enable.sparse.connections=true` configuration property
```

Comment:
maybe: "To alleviate this, the `enable.sparse.connections` configuration property can
be set to `true`, in which case the client will only connect to brokers it needs
to commnicate with, and only when necessary."